lib/test_summary_buildkite_plugin/input.rb

```diff
@@ -78,6 +77,14 @@ def job_id_regex
           DEFAULT_JOB_ID_REGEX
         end
       end
+
+      def handle_error(err)
+        if fail_on_error
+          raise
```

Comment:
```suggestion
          raise err
```
Not really related to your PR, but I wonder why we were not re-raising the original error?

Reply:
Using `raise` without a parameter automatically re-raises the "current" exception (i.e. `$!`). Still, I've accepted your suggestion, since it makes things clearer.